code/modules/assembly/assembly.dm

```diff
@@ -24,28 +24,17 @@
 	var/wires = ASSEMBLY_WIRE_RECEIVE | ASSEMBLY_WIRE_PULSE
```

Comment:
Can you change cooldown to `var/cooldown = 0` 
It probably got changed log ago by mistake.